Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js

```diff
@@ -3,17 +3,40 @@
 // Licensed under the MIT License
 // *********************************************************************
 export const inputModeEnum = {
-    streaming: 'streaming'
+    streaming: 'streaming',
+    batching: 'batching'
+};
+
+export const batchIntervalTypeEnum = {
+    day: 'day',
+    hour: 'hour',
+    min: 'min'
 };
 
 export const inputTypeEnum = {
     events: 'events',
     iothub: 'iothub',
     kafka: 'kafka',
     kafkaeventhub: 'kafkaeventhub',
+    blob: 'blob',
     local: 'local'
 };
 
+export const inputCompressionTypeEnum = {
+    none: 'none',
+    gzip: 'gzip'
+};
+
+export const jobModeEnum = {
+    recurrence: 'recurrence',
+    nonrecurrence: 'nonrecurrence',
+};
+
+export const inputFormatTypeEnum = {
+    json: 'json',
+    parquet: 'parquet'
```

Comment:
did you actually add support for parquet or is this work still pending? If not supported or soon to be supported shortly afterwards, I would recommend you to remove it until the support is about to exist.

Reply:
nevermind, I see you don't expose it to the UI dropdown, this is just a enum. Ignore what I said. You did the right thing here.

Reply:
cool